modules/ROOT/pages/managing-extension-studio.adoc

```diff
@@ -5,7 +5,7 @@
 The Bonita platform is highly extensible so we created a dedicated view to manage the extensions of your project
 
 == Opening the extensions view
-The extensions view is opened from the Studio cool bar.
+The extensions view is opened from either the new button in the Studio cool bar or from the project overview.
```

Comment:
It feels like you are talking about a New button (one that was not there before), the 'the' new button ^^'

```suggestion
The extensions view is opened from either the project overview or from the coolbar _New -> extension_.
```